ztp/argocd/deployment/resource-hook-role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Please move all the files under ztp/argocd/  to ztp/gitops-subscriptions/argocd

Reply:
fixed